setup.py

```diff
@@ -98,15 +98,20 @@
                     os.path.relpath(root, PACKAGENAME), filename))
 package_info['package_data'][PACKAGENAME].extend(c_files)
 
+setup_requires = metadata.get('requires-dist')
+if setup_requires is None:
+    setup_requires =  ['numpy', 'cython', 'astropy']
+else:
+    # Convert the config setup_requires string into a list
+    setup_requires = setup_requires.strip().split()
 # Note that requires and provides should not be included in the call to
 # ``setup``, since these are now deprecated. See this link for more details:
 # https://groups.google.com/forum/#!topic/astropy-dev/urYO8ckB2uM
-
 setup(name=PACKAGENAME,
       version=VERSION,
       description=DESCRIPTION,
       scripts=scripts,
-      install_requires=['astropy'],
+      setup_requires=setup_requires,
```

Comment:
I think this issue is unique to packages that rely on Cython (e.g. astroscrappy). If cython is not included in setup_requires, then the build will obviously not work because it can not generate the c source files. To get the numpy linking correct, often numpy.get_include() is used, which also requires numpy to be in setup_requires.
Upon further inspection, I believe that astropy does not need to be in setup_requires. 

My thought would be to do something like:
setup_requires = ['numpy', 'cython']
install_requires = install_requires
where install_requires is from the setup.cfg file. What do you think? Is there a cleaner way to fix the cython issue?